java/daikon/QuantBody.java.jpp

```diff
@@ -2186,6 +2186,7 @@ The "defined(INT)" test ensures that it only appears once in the output.
     assert (fields != null);
     assert (fieldsStartIdx >= 0 && fieldsStartIdx < fields.length);
 
+    @SuppressWarnings("signedness:cast.incomparable")
```

Comment:
Note that this suppression has no effect: it would only suppress something from this declaration, but that's not what you want here, as there is no cast.

Reply:
Thanks, already addressed.